app/services/bills_actions_service.rb

```diff
@@ -0,0 +1,147 @@
+class BillsActionsService
+  module ActionTypes
+    INTRODUCED = %w(
+      bill:filed
+      bill:introduced
+      committee:referred
+      governor:received
+    )
+
+    PASSED = %w(
+      committee:passed
+      committee:passed:favorable
+      committee:passed:unfavorable
+      bill:passed
+      governor:signed
+      bill:veto_override:passed
+    )
+    
+    FAILED = %w(
+      committee:failed
+      bill:failed
+      governor:vetoed
+      governor:vetoed:line-item
+      bill:veto_override:failed
+    )
+
+    COMPLETED = [*PASSED, *FAILED]
+  end
+
+  module StageTypes
+    COMMITTEE = %w(
+      committee:referred
+      committee:passed
+      committee:passed:favorable
+      committee:passed:unfavorable
+      committee:failed
+    )
+
+    GOVERNOR = %w(
+      governor:received
+      governor:signed
+      governor:vetoed
+      governor:vetoed:line-item
```

Comment:
ditto here, is this important to determine bill state right?

Reply:
Again, this is just to list all the possibilities so we can boil them down into values we care about.